src/settings-ui/Settings.UI/Views/VideoConference.xaml.cs

```diff
@@ -19,17 +18,30 @@ public sealed partial class VideoConferencePage : Page, IRefreshablePage
 
         private static async Task<string> PickFileDialog()
         {
-            FileOpenPicker openPicker = new FileOpenPicker();
-            openPicker.ViewMode = PickerViewMode.Thumbnail;
-            openPicker.SuggestedStartLocation = PickerLocationId.PicturesLibrary;
-            openPicker.FileTypeFilter.Add(".jpg");
-            openPicker.FileTypeFilter.Add(".jpeg");
-            openPicker.FileTypeFilter.Add(".png");
-            var hwnd = WinRT.Interop.WindowNative.GetWindowHandle(App.GetSettingsWindow());
-            WinRT.Interop.InitializeWithWindow.Initialize(openPicker, hwnd);
-
-            StorageFile file = await openPicker.PickSingleFileAsync();
-            return file?.Path;
+            // this code was changed to solve the problem with WinUI3 that prevents to select a file
+            // while running elevated, when the issue is solved in WinUI3 it should be changed back
+            OpenFileName openFileName = new OpenFileName();
+            openFileName.StructSize = Marshal.SizeOf(openFileName);
+            openFileName.Filter = "Images (*.jpg, *.jpeg, *.png)\0*.jpg; *.jpeg; *.png\0";
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/api/commdlg/ns-commdlg-openfilenamea

says `Do not include spaces in the pattern string.` . Although it works good as it is, let's remove spaces from pattern string just to be safe

Reply:
Missed this one, I'll fix it.